editor/components/block-list/index.js

```diff
@@ -62,15 +70,45 @@ class BlockList extends Component {
 		document.addEventListener( 'copy', this.onCopy );
 		document.addEventListener( 'cut', this.onCut );
 		window.addEventListener( 'mousemove', this.setLastClientY );
+		// Needs document to also work in inspector.
+		// In other words, if there is selection, but no focus.
+		document.addEventListener( 'keydown', this.onKeyDown );
```

Comment:
Is it viable to use the `KeyboardShortcuts` component instead?

https://github.com/WordPress/gutenberg/tree/master/components/keyboard-shortcuts

Might also help with the platform normalization:

>Mousetrap 1.4 introduced a generic mod helper which lets you set cross platform shortcuts.
>
>```
>Mousetrap.bind('mod+s', _saveDraft);
>```
>
>On Mac this ends up mapping to command+s whereas on Windows and Linux it maps to ctrl+s.

https://craig.is/killing/mice

Reply:
How do we dynamically change those? Rebind as the selection changes? There's also no concept of `access` in Mousetrap, unlike TinyMCE.

Reply:
Hm, we're also not using this component anywhere else, so I'm wondering why now.

Reply:
>Hm, we're also not using this component anywhere else, so I'm wondering why now.

We are:

https://github.com/WordPress/gutenberg/blob/2e276addb9bbf0f511eba55d12bf84c02b1b5fc7/editor/components/navigable-toolbar/index.js#L60-L66

https://github.com/WordPress/gutenberg/blob/2e276addb9bbf0f511eba55d12bf84c02b1b5fc7/editor/edit-post/modes/keyboard-shortcuts/index.js#L33-L35